### PR TITLE
Record app health heartbeat + default ExtensionEvent OperationId to VMWatchCohortId

### PR DESCRIPTION
**Pull request by frank-pang-msft**
Record app health heartbeat every 5 mins + default ExtensionEvent OperationId to VMWatchCohortId if VMWatchSettings specifies it in EnvironmentAttributes.

### CODE REVIEW

File: internal/telemetry/telemetry.go [hunk 1]
```diff
@@ -60,6 +61,9 @@ func NewTelemetry(h *handlerenv.HandlerEnvironment) (*Telemetry, error) {
 		instance = &Telemetry{
 			eem: extensionevents.New(logging.NewNopLogger(), &h.HandlerEnvironment),
 		}
+		// OperationId is initialized here but currently AppHealth telemetry does not depend on it.
+		// There are other scenarios for VMWatch where it is overridden
+		instance.eem.SetOperationID(uuid.New().String())
```
Comment (frank-pang-msft) [inline internal/telemetry/telemetry.go:66]: @zmyzheng 
It seems we never initialized OperationId for Linux, for parity with Windows I added this. 

I could remove it entirely.

File: main/cmds.go [hunk 2]
```diff
@@ -132,6 +133,15 @@ func enable(lg *slog.Logger, h *handlerenv.HandlerEnvironment, seqNum uint) (str
 		telemetry.SendEvent(telemetry.InfoEvent, telemetry.AppHealthTask, fmt.Sprintf("Grace period set to %v", gracePeriodInSeconds))
 	}
 
+	// Try to set VMWatchCohortId as extension event operationId
+	if vmWatchSettings != nil {
+		if vmWatchCohortId, err := vmWatchSettings.TryGetVMWatchCohortId(); err != nil {
+			telemetry.SendEvent(telemetry.ErrorEvent, telemetry.AppHealthTask, "Error while getting VMWatchCohortId", "error", err)
```
Comment (klugorosado) [inline main/cmds.go:139]: use Telemetry.SetupVMWatchTask or another TaskName

File: main/cmds.go [hunk 3]
```diff
@@ -151,6 +161,13 @@ func enable(lg *slog.Logger, h *handlerenv.HandlerEnvironment, seqNum uint) (str
 	//	1. Grace period expires, then application will either be Unknown/Unhealthy depending on probe type
 	//	2. A valid health state is observed numberOfProbes consecutive times
 	for {
+		// Since we only log health state changes, it is possible there will be no recent logs for app health extension.
```
Comment (klugorosado) [inline main/cmds.go:164]: This will work; however, I would create a go routine with the time ticker to log the event. I will be more consistent

**Review by klugorosado — CHANGES_REQUESTED** [review]

File: main/cmds.go (hunk 2 above)
Comment (klugorosado) [reply main/cmds.go]: nevermind

**Review by klugorosado — APPROVED** [review]

**Review by zmyzheng — APPROVED** [review]

**Pull request merged**